.github/workflows/hot_fix_pr.yaml

```diff
@@ -0,0 +1,55 @@
+---
+name: Hot Fix PR
+on:
+  workflow_dispatch:
+    inputs:
+      pr-number:
+        description: PR number to hot fix (must be merged into development branch)
+        type: string
+        required: true
+      dry-run:
+        description: |
+          Dry run: Print out the changes that would be made but do not create the PRs
+        type: boolean
+        required: true
+        default: false
+  issue_comment:
+    types: [created]
+jobs:
+  hot-fix-pr:
+    name: Hot Fix PR
+    runs-on: ubuntu-latest
+    # Only run on manual dispatch or when comment is "/hotfix" on a PR
+    if: |
+      github.event_name == 'workflow_dispatch' || 
+      (github.event_name == 'issue_comment' && 
+       github.event.issue.pull_request != null && 
+       github.event.comment.body == '/hotfix')
+    steps:
+      - name: Checkout code
```

Comment:
The script clones the repo. And before doing so, it just does some githu api calls. Is this checkout used for anything?

Reply:
I'm pretty sure you can't access the script without cloning the repo first, no?

Reply:
Oh, right, I didn't think of that :)